dashboard/app/models/ability.rb

```diff
@@ -30,7 +30,14 @@ def initialize(user)
         District,
         Workshop,
         Cohort,
-        WorkshopAttendance
+        WorkshopAttendance,
+        # PLC Stuff
+        ProfessionalLearningCourse,
+        ProfessionalLearningModule,
+        ProfessionalLearningTask,
+        PlcEvaluationQuestion,
+        PlcEvaluationAnswer,
+        UserProfessionalLearningCourseEnrollment
```

Comment:
Same feedback as Laurel above -- I'd like to see the naming resolved before merge.